packages/editor-ui/src/components/NodeSettings.vue

```diff
@@ -281,6 +343,42 @@ export default mixins(
 			},
 		},
 		methods: {
+			/**
+			 * Check if any of the node's credential types may be used to
+			 * make a request with the HTTP Request node v2.
+			 */
+			checkHttpRequestNodeSupport(credentialTypeNames: string[]) {
+				this.isSupportedByHttpRequestNode = credentialTypeNames.some(name => {
+					const credentialType = this.getCredentialTypeByName(name);
+
+					return (
+						credentialType.name.slice(0, -4).endsWith('OAuth') ||
```

Comment:
this is not a great way of checking this... instead we should check if credential type extends OAuth https://github.com/n8n-io/n8n/blob/master/packages/editor-ui/src/components/CredentialEdit/CredentialEdit.vue#L300